pkgs/development/python-modules/natural/default.nix

```
@@ -0,0 +1,25 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+}:
+buildPythonPackage rec {
+  pname = "natural";
+  version = "0.2.0";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "tehmaze";
+    repo = "natural";
+    rev = "${version}";
```

```suggestion
    rev = "refs/tags/${version}";
```
Helps to avoid branch/tag confusion.